templates/docs/INDEX.md

```diff
@@ -1,13 +1,63 @@
 # Templates
 
-A list of all template repos
+Highlighting a few different categories of templates
 
-⭐Retrieval Augmented Generation Chatbot: Build a chatbot over your data. Uses OpenAI and Pinecone.
+## ⭐ Popular
```

Comment:
IMHO, Icons with headers are not helpful. They make reading comprehension harder. Yeah, it looks fancy but less practical. 
There are methods to make pages look beautiful, and they do not harm readiness.